src/main/java/mate/academy/spring/dto/ShoppingCartDto.java

```diff
@@ -0,0 +1,38 @@
+package mate.academy.spring.dto;
+
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartDto {
+    private Long id;
+    private List<Ticket> tickets;
```

Comment:
```suggestion
    private List<Long> ticketIds;
```
dont return entities, rather create a dto for ticket or use just id 
id is ok

Reply:
Yes, You are right